src/db/spanner/models.rs

```diff
@@ -1527,7 +1534,12 @@ impl SpannerDb {
         };
         sqlparams.insert(
             "ids".to_owned(),
-            as_list_value(params.bsos.iter().map(|pbso| pbso.id.clone())),
+            params
+                .bsos
+                .iter()
+                .map(|pbso| pbso.id.clone())
+                .collect::<Vec<String>>()
```

Comment:
Since we can't call `to_spanner_value()` on an iterator, we have to collect the iterator first. I think this may result in a performance hit, but I'm not sure how significant it would be